clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -20,8 +20,10 @@ import org.elasticsearch.client.RestClient
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Success
 
+import java.util.UUID
+
 class HttpElasticsearchDAO private[dataaccess] (
-    private[dataaccess] val httpHosts: Seq[HttpHost]
+  private[dataaccess] val httpHosts: Seq[HttpHost]
```

Comment:
Wish there was an ignore whitespace option in this thing to work around scalafmt.

Reply:
Adding `?w=1` to the URL when viewing files should do it. Not sure why Github hasn't added it to the UI.